src/index.ts

```diff
@@ -341,17 +341,26 @@ const oauth2 = <Profiles extends string>({
         return {
           async authorized(...profiles: Profiles[]) {
             for (const profile of profiles) {
+              if (profile === 'twitch') {
```

Comment:
not compatible for everyone

Reply:
the `profile` does not have to be named "twitch". it can be named anything you want.

Reply:
Fixed.